x/programs/runtime/map.go

```diff
@@ -23,10 +23,10 @@ type maps map[string][]byte
 
 // Key value store for program data
 type storage struct {
-	// uint64 for simplicity, could be a real hash later
-	state   map[uint64]maps
-	mods    map[uint64]api.Module
-	counter uint64
+	// int64 for simplicity, could be a real hash later
```

Comment:
using int64 to avoid overflow errors(WASM uses ints). 
Also this needs to be changed anyways in the actual vm